### ISSUE TITLE
has this repo reached MVP status?

### ISSUE THREAD

**Issue by brianzelip** [NCBI-Hackathons/iCN3D-MMTF#4]
Has a minimum viable product shipped here?

I ask because the [test page linked from the readme](http://www.ncbi.nlm.nih.gov/Structure/icn3d2/test.html?mmtfid=1kq2) isn't working, and am wondering if this should be included in the list of products shipped from NCBI hackathons.

**Comment by jiywang3** [comment]
Yes, it's in production already. I will update the url in a min.
